lib/improver/nowcasting/forecasting.py

```diff
@@ -324,3 +327,101 @@ def grid_spacing(coord):
 
         advected_cube = amend_metadata(advected_cube, **self.metadata_dict)
         return advected_cube
+
+
+class CreateExtrapolationForecast(object):
+    """
+    Class to create a nowcast extrapolation forecast using advection.
+    For precipitation rate forecasts orographic enhancement must be used.
+    """
+
+    def __init__(self, input_cube, vel_x, vel_y,
+                 orographic_enhancement_cube=None, metadata_dict=None):
+        """
+        Initialises the plugin.
```

Comment:
This is not strictly a plugin as it doesn't use a process method. 
```suggestion
        Initialises the object.
```